service/dap/server.go

```diff
@@ -97,26 +97,26 @@ var defaultArgs = launchAttachArgs{
 
 // DefaultLoadConfig controls how variables are loaded from the target's memory,
 // borrowing the default value from the original vscode-go debug adapter and rpc server.
-// For evaluate requests made in the context of 'repl' or 'variables', they are triggered
-// by explicit human action so we use InteractiveEvaluateLoadConfig to apply larger limits.
 // TODO(polina): Support setting config via launch/attach args or only rely on on-demand loading?
-var (
-	DefaultLoadConfig = proc.LoadConfig{
-		FollowPointers:     true,
-		MaxVariableRecurse: 1,
-		MaxStringLen:       64,
-		MaxArrayValues:     64,
-		MaxStructFields:    -1,
-	}
+var DefaultLoadConfig = proc.LoadConfig{
+	FollowPointers:     true,
+	MaxVariableRecurse: 1,
+	MaxStringLen:       64,
+	MaxArrayValues:     64,
+	MaxStructFields:    -1,
+}
 
-	InteractiveEvaluateLoadConfig = proc.LoadConfig{
-		FollowPointers:     true,
-		MaxVariableRecurse: 64,
-		MaxStringLen:       4096,
-		MaxArrayValues:     4096,
-		MaxStructFields:    -1,
-	}
-)
+// For evaluate requests made in the context of 'repl' or 'variables', they are triggered
+// by explicit human action and the results can be displayed in a larger space (e.g. DEBUG CONSOLE)
```

Comment:
I think the reason is not the space, but the fact that this is a one-off request, not something that might be automatically triggered many times every time a breakpoint is hit. The user will know that the variable is large and be tolerant of the performance implications in this case.

Reply:
updated.